onnxruntime/core/framework/allocation_planner.cc

```diff
@@ -177,6 +177,12 @@ class PlannerImpl {
   }
   int& UseCount(const OrtValueName& name) { return UseCount(Index(name)); }
 
+  int DecrementUseCount(OrtValueIndex n) {
+    int& use_count = --UseCount(n);
```

Comment:
static

Reply:
This calls the non-static UseCount method, so cannot be static